homeassistant/components/automation/blueprints/motion_light.yaml

```diff
@@ -36,23 +36,23 @@ mode: restart
 max_exceeded: silent
 
 trigger:
```

Comment:
If we update the syntax, we should update it fully and also pularize (is that a word?) the trigger, action & condition top level keys as well.

Reply:
Good point. Updated the PR.